unrpyc.py

```diff
@@ -47,87 +47,12 @@ def acquire(self, block=True, timeout=None):
         def release(self):
             pass
 
+
```

Comment:
Try to avoid random changes like this if you can, unless they fix adherence to the used coding style. It clutters the blame.